<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::MimeTypesWidget</class>
 <widget class="QWidget" name="GammaRay::MimeTypesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="searchLine"/>
   </item>
   <item>
    <widget class="QTreeView" name="mimeTypeView">
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
